dvc/utils/__init__.py

```diff
@@ -369,3 +368,10 @@ def _visual_center(line, width):
     right_padding = spaces - left_padding
 
     return (left_padding * " ") + line + (right_padding * " ")
+
+
+def relpath(path, start=os.curdir):
+    # Windows path on different drive than curdir doesn't have relpath
+    if os.name == "nt" and not os.path.commonprefix([os.getcwd(), path]):
```

Comment:
Using `os.getcwd()` is an error here I believe, it should be `os.path.abs(start)` instead. Now this will fail:
```python
relpath("C:\\a\\b", "D:\\")
```
if curdir is on `C:`. I suggest adding `start = os.path.abs(start)` at the start of the function and then use it everywhere.